lib/router/src/router.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+
+import { Link, Location, navigate, LocationProvider } from '@reach/router';
+import { ToggleVisibility } from './visibility';
+import { queryFromString, storyDataFromString } from './utils';
+
+interface RenderData {
+  path: string;
+  location: object;
+  navigate: (to: string) => void;
+  viewMode?: string;
+  storyId?: string;
+}
+interface MatchingData {
+  match: null | { path: string };
+}
+
+interface QueryLocationType {
```

Comment:
If an interface describes prop or state - add it as a suffix. So you will differentiate interfaces of objects/models and interfaces which describes communications between components.

Reply:
✅

Reply:
Should I place the interfaces in a different file?

Reply:
If you plan to export interfaces and reuse them in other source files - then "yes", otherwise - if an interface is a private to a component - no need to move.